receiver/zipkinreceiver/trace_receiver.go

```diff
@@ -217,7 +217,8 @@ func (zr *ZipkinReceiver) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	transportTag := transportType(r, asZipkinv1)
 	ctx = obsreport.ReceiverContext(ctx, zr.id, transportTag)
-	ctx = obsreport.StartTraceDataReceiveOp(ctx, zr.id, transportTag)
+	rec := obsreport.NewReceiver(obsreport.ReceiverSettings{ReceiverID: zr.id, Transport: transportTag})
```

Comment:
This can be cached in the receiver struct.

Reply:
Done in latest commit